src/EditorFeatures/TestUtilities/Workspaces/TestWorkspace_XmlConsumption.cs

```diff
@@ -280,7 +280,8 @@ public static TestWorkspace Create(string xmlDefinition, bool completed = true,
             }
             else
             {
-                filePath = projectName +
+                // Prepend "./" because DocumentState expects the project to be within a directory when getting AnalyzerOptions.
```

Comment:
This is correctly fixed in https://github.com/dotnet/roslyn/pull/39463/commits/2cbd65a304603cf897805a8888eed2c32cc427d3

Reply:
reverted